services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/core.py

```diff
@@ -40,6 +40,7 @@
 
 logger = logging.getLogger(__name__)
 CONTAINER_WAIT_TIME_SECS = 2
+_TASK_PROCESSING_PROGRESS_WEIGHT: Final[float] = 0.99
```

Comment:
If I understand this well, this weight affects the actual progress and you expect to be constraint. I encourage to document this with asserts in here in case some clamsy developer like myself changes this unexpectedly


For instance
```python

assert 0.8<=_TASK_PROCESSING_PROGRESS_WEIGHT<1 # nosec
```

Reply:
added assertion. thanks